Doc/library/sqlite3.rst

```diff
@@ -2358,6 +2345,88 @@ can be found in the `SQLite URI documentation`_.
 .. _SQLite URI documentation: https://www.sqlite.org/uri.html
 
 
+.. _sqlite3-howto-row-factory:
+
+How to create and use row factories
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, :mod:`!sqlite3` represents each fetched row as a :class:`tuple`.
+If a :class:`!tuple` does not suit your needs,
+use the :class:`sqlite3.Row` class or a custom :attr:`~Cursor.row_factory`.
+
+Even though :attr:`!row_factory` exists as an attribute both on the
+:class:`Cursor` and the :class:`Connection`,
+it is recommended to set it as :class:`Connection.row_factory`,
+so all cursors created off of the connection will use the same row factory.
```

Comment:
```suggestion
so all cursors created from the connection will use the same row factory.
```

Like above, "from" is a bit clearer and more direct IMO, particularly I'd think for non-native-level speakers